.config/typos.toml

```diff
@@ -14,6 +14,9 @@ exat = "exat"
 optin = "optin"
 smove = "smove"
 
+[default.ignore-words]
+"Parth"
```

Comment:
I don't know the Toml syntax but this appears to follow the pattern used for other rules.

```suggestion
parth = "Parth"
```

Reply:
Yeah, I also don't see ignore-words as one of the options supported on https://github.com/crate-ci/typos/blob/master/docs/reference.md

Reply:
Well now it passed...

Reply:
let me try to push the typo and test it

Reply:
No. It didn't work :(